ansible/roles/baselayout/tasks/ccache.yml

```diff
@@ -8,7 +8,7 @@
   # This depends on ansible being able to run curl locally, YMMV, if it doesn't
   # work, try changing the local_action to a remote one:
   #  raw: curl -sL https://www.samba.org/ftp/ccache/
-  local_action: command shell -sL https://www.samba.org/ftp/ccache/
+  local_action: command shell curl -sL https://www.samba.org/ftp/ccache/
```

Comment:
removed in #1723, hopefully by accident, also I discovered that smartos is shipping with a recent enough ccache now so we don't even take this path for smartos18

Reply:
Was in #1735, and was unintentional, sorry, I was flailing around trying to get the wget to work.

Reply:
ohhh, yeah, been there with `wget`, sorry for your pain